packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -8,8 +8,9 @@ Terra.describeViewports('Date Picker', ['tiny', 'small', 'medium'], () => {
     });
 
     it('sets the date', () => {
-      browser.setValue('input[name="terra-date-date-input"]', '06/01/2017');
```

Comment:
This is how some tests currently set values in the input by looking up the `name` attribute. These changes they would now have to set values into each individual input and have to know what the name attributes are. It would be helpful for consumers to document things like this in [here](https://engineering.cerner.com/terra-framework/components/terra-date-picker/date-picker/date-picker#testing).

Reply:
Added the documentation here : https://github.com/cerner/terra-framework/pull/1208/commits/ef6e9b4856db01f837879b86710bbe32a85e6a66